### PR TITLE
created product page

### PR DESCRIPTION
**Pull request by agrawalshanu708**
created product page
made two section one for filter and product showcase
Screenshot (87)
Screenshot (88)

### CODE REVIEW

**Review by therajatg — APPROVED** [review]
looks clean

**Review by zabihshaik — APPROVED** [review]
Everything looks good.
UI looks amazing.

**Comment by agrawalshanu708** [comment]
thanx guyzz

File: product/product.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+.img-box img{
+    width: 100%;
+    height: 100%;
+}
+
+.wishlist-icon{
+   position: absolute;
+   top:2rem;
+   right: 2rem;
+   color: red;
```
Comment (yashpatel1797) [inline product/product.css:89]: here instead of assigning value use variable.

**Review by yashpatel1797 — COMMENTED** [review]
Looks good to merge.

File: product/product.css (hunk 1 above)
Comment (agrawalshanu708) [reply product/product.css:89]: thanx yash, I will do it

**Pull request merged**